manual/_static/eqno.js

```
@@ -0,0 +1,52 @@
+      parentNode.insertBefore(document.createTextNode(beforeMatch), node);
+      parentNode.insertBefore(span, node);
+      const afterMatch = text.substring(regex.lastIndex);
+      parentNode.insertBefore(document.createTextNode(afterMatch), node);
+
+      // Remove the original text node.
+      parentNode.removeChild(node);
+
+      // Continue with the next match.
+      text = afterMatch;
+      regex.lastIndex = 0;
+      node = parentNode.lastChild;
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)